Mogakco/Sources/Data/DataMapping/UpdateUserIDRequestDTO.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  UpdateUserIDRequestDTO.swift.swift
+//  Mogakco
+//
+//  Created by 신소민 on 2022/11/26.
+//  Copyright © 2022 Mogakco. All rights reserved.
+//
+
+import Foundation
+
+struct UpdateUserIDRequestDTO: Encodable {
+    private let userIDs: ArrayValue<StringValue>
```

Comment:
```suggestion

struct UpdateUserIDsRequestDTO: Encodable {
    private let userIDs: ArrayValue<StringValue>
```
API명과 맞춰 복수형으로 이름을 가지면 좋을 것 같아요!

Reply:
여기도 단수형으로 썼네요. 리뷰 반영했습니다. 👏 👏